docs-ref-services/hdinsight.md

```diff
@@ -1,29 +1,391 @@
 ---
-title: Azure HDInsight libraries for Java
-description: 
+title: Azure HDInsight Java SDK
+description: Reference for Azure HDInsight Java SDK
 keywords: Azure, Java, SDK, API, HDInsight
-author:  
-ms.author:  
-manager:  
-ms.date:  
+author: tylerfox
+ms.author: tyfox
+manager: arindamc
+ms.date: 9/20/2018
 ms.topic: article
 ms.prod: azure
 ms.technology: azure
 ms.devlang: java
 ms.service: hdinsight
 ---
 
-# Azure HDInsight libraries for Java
-
+# HDInsight Java Management SDK Preview
 ## Overview
+The HDInsight Java SDK provides classes and methods that allow you to manage your HDInsight clusters. It includes operations to create, delete, update, list, scale, execute script actions, monitor, get properties of HDInsight clusters, and more.
+
+## Prerequisites
+* An Azure account. If you don't have one, [get a free trial](https://azure.microsoft.com/free/).
+* [Java JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Maven](https://maven.apache.org/install.html)
+
+## SDK Installation
+The HDInsight Java SDK is available through Maven [here](https://mvnrepository.com/artifact/com.microsoft.azure.hdinsight.v2018_06_01_preview/azure-mgmt-hdinsight). Add the dependency to your pom.xml.
```

Comment:
Can this be a code snippet of the `<dependency>` element for `pom.xml`? The way this paragraph is structureds, seems that the two links below are more important than the `here` hyperlink, that is more likely to be missed by the reader.